mscore/seq.cpp

```diff
@@ -562,6 +562,14 @@ void Seq::processMessages()
                         break;
                   case SeqMsgId::PLAY:
                         putEvent(msg.event);
+                        // Update initList immediately if there're some changes in mixer
```

Comment:
@lasconic Yes, you're right. I was confused because seq->setControler() enqueues cross-thread calls... but in this case it doesn't make sence. Thanks.